src/utils/functions/render-program-section/renderProgramSection.test.tsx

```
@@ -96,7 +106,7 @@ describe('renderProgramSection', () => {
     };
 
     beforeEach(() => {
-        Object.keys(capturedProps).forEach((k) => delete capturedProps[k]);
+        Object.keys(mockCapturedProps).forEach((k) => delete mockCapturedProps[k]);
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)